developer-experience/recipes/remote-debugging/kubernetes-dashboard/kubernetes-dashboard.md

```diff
@@ -0,0 +1,42 @@
+# Remote Debugging using the Kubernetes Dashboard
+
+This document covers the Kubernetes Dashboard which is a useful tool for remote debugging for Kubernetes Clusters. It allows the management of applications running in the cluster, debug them and manage the cluster all through this dashboard.
+
+## Overview and Background
+
+Developer experience focuses on how easy or difficult it is for a developer to perform essential tasks needed to implement a change including build, test, start, and debug.
+
+There are times when not all solutions can be run locally. This limitation could be due to a cloud service which does not offer a robust or efficient way to locally debug the environment. In these cases, it is necessary to deploy use other tools that enabling this remote debugging. One such example is the Kubernetes dashboard.
+
+## Pre-Requisites
+
+- Kubernetes cluster up and running
+- [Install Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/): command-line tool that allows you to run commands against Kubernetes clusters.
+
+## Advantages and Use Cases
+
+- Allows the ability to view, manage and monitor the operational aspects of the Kubernetes Cluster.
+
+- Provides an easy to use web-based UI to be able to do do the following:
+  - See an overview of the cluster
+  - deploy applications onto the cluster
```

Comment:
maybe not worth mentioning at this list, but I found it amazing that you see the pod state real time, as they are being terminated or started etc.

Reply:
yes, agreed. Added